sphinx_rtd_theme/__init__.py

```diff
@@ -55,4 +55,7 @@ def setup(app):
         app.add_message_catalog('sphinx', rtd_locale_path)
         app.connect('config-inited', config_initiated)
 
+    # sphinx emits the permalink icon for headers, so choose one more in keeping with our theme
+    app.config.html_permalinks_icon = "🔗"
+
```

Comment:
This should fix the tests from failing
```
    if sphinx_version >= (3, 5, 0):
        app.config.html_permalinks_icon = "🔗"
    else:
        app.config.html_add_permalinks = "🔗"
   ```

Reply:
Nice, did that. Also specified the unicode character with `"\u..."`. Hope that kind of string literal is portable across python versions.